query_optimizer/ExecutionHeuristics.cpp

```diff
@@ -101,5 +104,23 @@ void ExecutionHeuristics::optimizeExecutionPlan(QueryPlan *query_plan,
   }
 }
 
+void ExecutionHeuristics::setBloomFilterProperties(serialization::BloomFilter *bloom_filter_proto,
+                                                   const CatalogRelation *relation) {
+  const std::size_t cardinality = relation->estimateTupleCardinality();
+  if (cardinality < kOneThousand) {
+    bloom_filter_proto->set_bloom_filter_size(kOneThousand / kCompressionFactor);
+    bloom_filter_proto->set_number_of_hashes(kVeryLowSparsityHash);
+  } else if (cardinality >= kOneThousand && cardinality < kTenThousand) {
```

Comment:
We could simplify to ` else if (cardinality < kTenThousand)`, as `cardinality >= kOneThousand` is always `true`.